### PR TITLE
Five lavaland ruins

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
The first of many ruins.

I'd rather not post pictures and spoil the exploration for people.

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
kor inspires me

**Comment by Ultimate-Chimera** [comment]
:+1:

**Comment by WJohn** [comment]
I was going to be doing these things but was lacking in basic assets. Now that we have diagonal walls, abductor stuff should be more viable. I'll see about working on that, but I'll need to talk to you in IRC.

File: code/modules/ruins/lavaland_ruin_code.dm [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+
+
+
+
+//lavaland_surface_seed_vault.dmm
+//Seed Vault
+/obj/item/seeds/gatfruit
+
```
Comment (KazeEspada) [inline code/modules/ruins/lavaland_ruin_code.dm]: What's this for?

Comment (KorPhaeron) [inline code/modules/ruins/lavaland_ruin_code.dm]: Coulda sworn I removed that

File: code/modules/ruins/lavaland_ruin_code.dm [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+//lavaland_surface_seed_vault.dmm
+//Seed Vault
+
+/obj/effect/spawner/lootdrop/seed_vault
+	name = "seed vault seeds"
+	lootcount = 1
+
+	loot = list(/obj/item/seeds/gatfruit = 10,
```
Comment (Iamgoofball) [inline code/modules/ruins/lavaland_ruin_code.dm]: :+1:

Comment (Cheridan) [inline code/modules/ruins/lavaland_ruin_code.dm]: this pleases gaga

**Pull request merged**